src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -1928,7 +1940,7 @@ class SignableCommand extends Command implements SignalableCommandInterface
 
     public function getSubscribedSignals(): array
     {
-        return [\SIGALRM];
+        return !\function_exists('pcntl_signal') || \in_array('pcntl_signal', explode(',', ini_get('disable_functions'))) ? [] : [\SIGALRM];
```

Comment:
```suggestion
        return SignalRegistry::isSupported() ? [\SIGALRM] : [];
```

(don't miss adding the use statement for `SignalRegistry`)